components/shipengine/sources/new-shipping-label/new-shipping-label.mjs

```diff
@@ -0,0 +1,52 @@
+import common from "../common/webhook.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  ...common,
+  key: "shipengine-new-shipping-label",
+  name: "New Shipping Label",
```

Comment:
Do you think the name could be changed to something such as `Shipping Label Created` instead?

This is to [adhere to the updated guidelines](https://pipedream.com/docs/components/guidelines/#component-key-pattern) and avoid duplicating names between actions and sources, as `New Shipping Label` could refer to either type of component (instead of e.g. `Create Shipping Label` for an action and `Shipping Label Created` for a source)